lua/core/comment.lua

```diff
@@ -19,7 +20,12 @@ function M.config()
 end
 
 function M.setup()
-  require("nvim_comment").setup(lvim.builtin.comment)
+  local nvim_comment = require "nvim_comment"
+
+  nvim_comment.setup(lvim.builtin.comment)
```

Comment:
can we separate `on_config_done` and `active` from the rest of the settings
```lua
lvim.builtin.comment = {
  active = true,
  on_config_done = nil
  opts = {
  },
}
nvim_comment.setup(lvim.builtin.comment.opts)
```

Reply:
Arf, there is a big style mix in our code base, I think we should go for this syntax as the other one would be more appropriate for defining lambdas. There was a quick discussion and that's what some of us agreed for (maybe only kylo and me actually :smile: ), but since it's not been smth official, you probably never heard of it.

Reply:
but what happens if one the plugins actually use this value 😅 
like for example `telescope` or `toggleterm` or whatever might actually accept `on_config_done` as an argument in their setup, what happens then 😢

Reply:
Sorry, I answered under the wrong comment :sweat: